CONTRIBUTING.md

```diff
@@ -31,6 +31,28 @@ The `vscodessh` subcommand on the `coder` binary periodically flushes it's netwo
 
 Coder Remote periodically reads the `network-info-dir + "/" + matchingSSHPID` file to display network information.
 
+## Testing
```

Comment:
I wanted this section below the development section, but when I do that Prettier goes haywire and starts injecting spaces all over the place, leading to insanity like this:

```
- Use the "VS Code Desktop" button from a Coder dashboard.
- Manually    open the link wit   h `Developer: Ope   n URL` from in
    sid   e VS Code.
    - Use    `code --    open-url    <
    link    >` on the     co   mm    and l    ine.

The        
        link format i    s `        v   sc    ode://co    der      
      .cod   er-    r        emote/    ope    n?$        {   qu    ery}
    `        .             For exam               ple:        
    
``     
       `bash
    cod               e                 --ope    n        -url    
             'vs            c           od    e        :/                /coder    
    .        cod                    e            r           -r    e        mo         
           te/ope        n                                            ?u        rl=               d                ev.cod    e                                    r            .c        om            &own                                   e        r=        my        -    u    s                    e        rn    am        e    &w                o               r    k    s                        pa    c    e                    =my    -                ws&    a            ge                                                                           nt=my-agent'
```

Reply:
I think all documentation should look like that